asset/css/site-nav.css

```diff
@@ -8,8 +8,9 @@
 #page-content {
     flex: 1;
     margin-left: 285px;
+    overflow-x: hidden;
```

Comment:
One of the disadvantage is that if your screen width is really small (e.g. 360px), no scroll bar will be shown, the content will be cut off and the user cannot see them. For example, this table's "Description" column cannot be seen at all: 

<img src="https://user-images.githubusercontent.com/3168908/42556081-02478400-851d-11e8-97b2-6ac98a0ab569.png" width="450" />

In our current MarkBind website, the scroll bar will be shown and user can scroll right:

<img src="https://user-images.githubusercontent.com/3168908/42556139-3cfcd1ea-851d-11e8-848a-3bea548a9b14.png" width="450" />